### DISCUSSION TITLE
cli: fake typing mode for `asciinema play`

### DISCUSSION THREAD

**Discussion by ku1ik** [asciinema/discussions#23] [Ideas]
As described in this tweet: 
https://twitter.com/ballingt/status/862820367663898624?cn=cmVwbHk%3D&refsrc=email

**Comment by AlekSi** [comment]
Something like [doitlive](https://doitlive.readthedocs.io/en/latest/)?

**Comment by ku1ik** [comment]
Jep, something like this.
